sampleDriver/EvsEnumerator.cpp

```diff
@@ -203,6 +205,16 @@ void EvsEnumerator::enumerateCameras() {
         }
     }
 
+    const std::string kCarlaSimReverse = "CarlaSimReverse";
+    const std::string kCarlaSimFront   = "CarlaSimFront";
+    const std::string kCarlaSimLeft    = "CarlaSimLeft";
+    const std::string kCarlaSimRight   = "CarlaSimRight";
+    sCameraList.emplace(kCarlaSimReverse, kCarlaSimReverse.c_str());
+    sCameraList.emplace(kCarlaSimFront, kCarlaSimFront.c_str());
+    sCameraList.emplace(kCarlaSimLeft,  kCarlaSimLeft.c_str());
+    sCameraList.emplace(kCarlaSimRight, kCarlaSimRight.c_str());
+    captureCount +=4;
```

Comment:
I prefer a comment here to describe that we have some kind static configuration/expectations here.